src/app/public/index.ts

```diff
@@ -1 +1,8 @@
-export * from './modules';
+export * from './modules/inline-form/inline-form-adapter.service';
+export * from './modules/inline-form/inline-form.component';
```

Comment:
Not sure we need to export this. Do you envision people pulling `SkyInlineFormComponent` into their component classes?

Reply:
Nope, that was an oversight. Removed.